src/BatteryReplacementCashFlowModel.py

```diff
@@ -265,4 +265,6 @@ def run(self, container, Inputs):
     # run the calculations, and compute NPV, IRR and PI
     metrics = main.run(settings, [component], {})
     for k, v in metrics.items():
+      if k == 'outputType':
```

Comment:
how do we handle k != 'outputType'? Otherwise I am not sure the full scope of this "if" condition.